air.toml

```diff
@@ -0,0 +1,13 @@
+[format]
+line-width = 100
+indent-width = 2
+indent-style = "space"
+line-ending = "auto"
+persistent-line-breaks = true
+exclude = [
+    # Syntax error bug of the formatter
+    # https://github.com/posit-dev/air/issues/255
+    "tests/testthat/test-as_polars_series.R",
+    "tests/testthat/test-expr-string.R",
+    "tests/testthat/test-datatypes-classes.R",
+]
```

Comment:
I'm confused, do you want to ignore those or remove raw strings as said in https://github.com/eitsupi/neo-r-polars/pull/73#issuecomment-2676138156?

Reply:
Sorry for the confusion. My intention was to ensure that the `task` command would succeed at this PR stage.

We can update the R files later to remove these lines, so I will add a TODO comment.

Reply:
Actually I had a patch ready for this in #73 so I can push it there and you can remove your lines here.